test/gtest/uct/ib/test_cq_moderation.cc

```diff
@@ -0,0 +1,179 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2001-2017.  ALL RIGHTS RESERVED.
+ *
+ * See file LICENSE for terms.
+ */
+
+extern "C" {
+#include <uct/api/uct.h>
+}
+#include <uct/uct_test.h>
+#include <ucs/time/time.h>
+#include <poll.h>
+#include <infiniband/verbs.h>
+
+#define TEST_PERIOD (3ul * UCS_USEC_PER_SEC) /* wait for 3 secs to get statistics */
+#define NSEC_PER_USEC (UCS_NSEC_PER_SEC / UCS_USEC_PER_SEC)
```

Comment:
there is UCS_NSEC_PER_SEC in ucs/time/time.h

Reply:
right, but here is value for number nsecs in usecs (micro / nano)

Reply:
right, sorry.